doc/api/http2.md

```diff
@@ -76,11 +76,12 @@ req.on('response', (headers, flags) => {
   }
 });
 
-let data = []
-req.on('data', (d) => data.push(d));
+req.setEncoding('utf8');
+let rawData = '';
+req.on('data', (chunk) => { rawData += chunk; });
 req.on('end', () => {
   console.log();
```

Comment:
Could we remove this extra `console.log` — I think it might confuse people as to why it's there.

Reply:
I'd like to have a newline between the headers and the body, since that will make it clearer which is which. Would it be better to put the newline as part of the next line, e.g. `console.log("\n" + ...)`?